dockerfiles/build.sh

```diff
@@ -34,7 +34,7 @@ build_directory() {
   shift
   # Calling build.sh
   if [ -e ${directory}/build.sh ] ; then
-    ${directory}build.sh ${OPTIONS} ${ARGS}
+    bash ${directory}build.sh ${OPTIONS} ${ARGS}
```

Comment:
why do we have bash there ? shebang is sh so we shouldn't require bash

Reply:
without this it was failed on build che server, idk why but it helped

Reply:
if build.sh is not compliant we should fix it instead of using bash there.

Reply:
I will recheck this

Reply:
@benoitf here is PR with proper fixes